### ISSUE TITLE
Private showcases

### ISSUE THREAD

**Issue by brew** [ckan/ckanext-showcase#23]
A status setting for showcases to make them private/public. This could help with drafting and approval processes.

**Comment by muhammetcetinyamac** [comment]
Indeed. Unfortunately, when we add a new item to the project gallery, it automatically becomes hidden. We can't change its visibility manually. I'm working on a solution for this.
